src/Currency/CurrencyAlpha3X.php

```diff
@@ -0,0 +1,24 @@
+<?php
+declare(strict_types=1);
+
+namespace PrinsFrank\Standards\Currency;
+
+/**
+ * @no-official-source
+ *
+ * @manually-updated
+ */
+enum CurrencyAlpha3X: string {
+    case EURX = 'EURX';
```

Comment:
I’m not sure if switching to 4 characters is a good idea, since the database uses CHAR(3). I can change it, but it looks like a backward compatibility issue.